src/web/utils/textHelpers.ts

```diff
@@ -26,8 +26,9 @@ export function formatUnixDate(timeValue: number) {
 
 export const separateStringsList = (strings: string): string[] => {
   if (strings === '') return [];
-  const stringsTrimmed = strings.replace(/, |,| {2}|\n|;/gi, ' ').trim();
-  return stringsTrimmed.split(/[ ]+/);
+  const stringsTrimmed = strings.replace(/, |,| {2}|\n|;|]|\[/gi, ' ').trim();
+  const ts = stringsTrimmed.split(/[ ]+/);
```

Comment:
This is because the result we are getting from admin is like a fake array? i.e. `"[abc.com, 123.com]"`? Could we just change admin so that it returns a proper string array? 

If not, perhaps we could rename `ts` to something that is a bit more clear? This method is now separating the string and trimming the brackets, maybe it deserves it's own method or a new name?

Reply:
Completely missed deleting this - I was just using it as a test at one point. Thank you for catching. I added the two brackets as replace values which I think still applies to this function in general.  I don't think we should change admin - the reviewer requested that it be in the format that it is, so it does error with the array on the backend.

Reply:
Put it back